web-common/src/features/dashboards/leaderboard/LeaderboardHeader.svelte

```diff
@@ -124,7 +130,7 @@
           on:click={toggleSortByActiveContextColumn}
           class="shrink flex flex-row items-center justify-end"
           aria-label="Toggle sort leaderboards by context column"
-          style:width={$widthPx}
+          style:width={widthPx}
```

Comment:
Are we ever using other units besides "px"? It'd feel better if we appended the "px" at the very end, and move around `widthPx` as a `number` rather than a `string`.